src/worldmap.test.ts

```diff
@@ -387,4 +376,46 @@ describe('Worldmap', () => {
     worldMap = new WorldMap(ctrl, document.getElementsByClassName('mapcontainer')[0]);
     worldMap.createMap();
   }
+
+  describe('when apply log is set', () => {
+    const LOW = 1;
+    const HIGH = 3000;
+    const AVG = (HIGH + LOW)/2;
+
+    beforeEach(() => {
+      ctrl.data = new DataBuilder()
+        .withCountryAndValue('SE', LOW)
+        .withCountryAndValue('IE', AVG)
+        .withCountryAndValue('US', HIGH)
+        .withThresholdValues([2])
+        .build();
+      ctrl.panel.circleMinSize = '2';
+      ctrl.panel.circleMaxSize = '10';
+      ctrl.panel.isLogScale = true;
+      ctrl.panel.logFunction = 'log1p';
+      worldMap.drawCircles();
+    });
+
+    it('should draw three circles on the map', () => {
+      expect(worldMap.circles.length).toBe(3);
+    });
+
+    it('should create a circle with min circle size for smallest value size', () => {
+      expect(worldMap.circles[0].options.radius).toBe(2);
```

Comment:
How the `circles` are ordered?

I would suggest to use `find` and use the country code rather relying on indexing.

In my opinion this logic is fragile, because it assumes ordering.

Reply:
Ah, yes, this comes from a data builder, where we have control over the creation of the circles. I also believe that using find would be nice, but I just used the default from other tests.